### ISSUE TITLE
All PHP Lambdas throwing 500 errors all of a sudden - nothing in logs?

### ISSUE THREAD

**Issue by homerjam** [vercel/vercel#2147]
These lambdas have been working up until recently (last night) with no explanation for why they're now broken...

😞 

https://mamag.img.earth/wp-admin
https://intmagic-nuxtpress.homerjam.now.sh/wp-admin

**Comment by anmonteiro** [comment]
@homerjam if this is the same issue that has been affecting me, using `--regions bru`, for example, might solve your issue. 

I reported this upstream to @rauchg and @matheuss and it's apparently a problem in the CDG region.

**Comment by homerjam** [comment]
Thanks for the info @anmonteiro - I'm specifying `bru1` region but will try another. FWIW this has been going on over 12 hours...

**Comment by homerjam** [comment]
Also I have js lambdas in that region that are working 🤷‍♂️

**Comment by ties-v** [comment]
Same issue, also with an alias with only  routes (now v1) and PHP deployment. 
Routed alias: demo.digitag.io PHP deployment (now v1 docker): demo.digitag.io/api or api.digitag.lt-it.cloud.

**Comment by matheuss** [comment]
@homerjam @ties-v investigating – will keep you posted

**Comment by fu-sen** [comment]
It may be improving because I look homerjam's link is normal.
(I'm in Japan so I mainly look at Tokyo server HND1)

It looks like a server-level issue, so expanding the topic in Now CLI issues may be an inappropriate place, 
but it's enough for ZEIT members to figure out and investigate.
If it's still going on, the ZEIT members will react soon.

**Comment by homerjam** [comment]
Yep - it's working again. It'd be great to know what happened - this isn't great for a production site.

**Comment by ties-v** [comment]
The PHP deployment itself is working (api.digitag.lt-it.cloud), but the Now v1 routing rules on demo.digitag.io are still serving a 500. 

Thanks for looking into it @matheuss !

**Comment by matheuss** [comment]
@homerjam we just finished deploying a fix for this. Yesterday we introduced a regression that affected deployments sending multiple headers with the same name. In your case, there were multiple `cache-control` headers (as per [RFC 2616 Section 4.2](https://tools.ietf.org/html/rfc2616#section-4.2), we merge them into one). It's not very common practice to do that, so it slipped through our tests. 

Alongside with the fix, we added multiple regression tests to ensure this doesn't happen again.

I'm closing this issue as it's fixed – @ties-v we're investigating what's up with your v1 path alias rules – ~I'll get back you here :+1:~ we spoke on Slack.
